task/oci-cli/0.1/oci-cli-task.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: oci-cli-task
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.54.0"
+    tekton.dev/categories: CLI
+    tekton.dev/tags: cli
+    tekton.dev/displayName: "oracle cli task"
+    tekton.dev/platforms: "linux/amd64"
+spec:
+  params:
+    - name: tenancy_ocid
+      description: "The OCID of the tenancy"
+    - name: user_ocid
+      description: "The OCID of the user"
+    - name: fingerprint
+      description: "The fingerprint of the API key"
+    - name: private_key
+      description: "The private key content (base64 encoded)"
+    - name: region
+      description: "The OCI region (e.g., us-ashburn-1)"
+    - name: command
+      description: "The OCI CLI command to execute"
+  steps:
+    - name: oci-cli
+      image: ghcr.io/oracle/oci-cli:latest
```

Comment:
will it be possible for you to use a specific tag?

Reply:
@vinamra28  done